hack/ocp-e2e-tests.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+# Make sure to set the IMAGE_REPO env variable to your quay.io username
```

Comment:
I don't see IMAGE_REPO used below. is this comment accurate? do the 'oc' commands consume it?

Reply:
IMAGE_REPO used in  Makefile [1] 

[1] 
https://github.com/openshift/kubernetes-nmstate/blob/master/Makefile#L4